demisto_sdk/commands/split/tests/ymlsplitter_test.py

```diff
@@ -252,7 +254,10 @@ def test_extract_image(tmpdir):
         assert image == DEFAULT_IMAGE_BASE64
 
 
-def test_extract_code(tmpdir):
+@pytest.mark.parametrize(
+    argnames="file_type", argvalues=[("integration"), ("betaintegration")]
+)
+def test_extract_code(tmpdir, file_type):
     """
     Given
     a unified integration file of python format.
```

Comment:
Let's add the additional use-case to dics
```suggestion
def test_extract_code(tmpdir, file_type):
    """
    Given
        Case 1: a unified integration file of python format.
        Case 1: a unified beta-integration file of python format.
```
Let's do this for all unit tests.